pkgs/development/tools/misc/gdb/default.nix

```diff
@@ -16,21 +16,19 @@
   ]
 }:
 
-let
-  basename = "gdb-${version}";
-  version = "9.2";
-in
-
 assert pythonSupport -> python3 != null;
 
 stdenv.mkDerivation rec {
+  pname = "gdb";
+  version = "9.2";
+
   name =
     stdenv.lib.optionalString (stdenv.targetPlatform != stdenv.hostPlatform)
                               (stdenv.targetPlatform.config + "-")
-    + basename;
+    + "${pname}-${version}";
 
   src = fetchurl {
-    url = "mirror://gnu/gdb/${basename}.tar.xz";
+    url = "mirror://gnu/gdb/${pname}-${version}.tar.xz";
```

Comment:
this will most likely cause a LOT of rebuilds, maybe save it for a different PR?

Reply:
Created #95875 .